routers/bird.php

```diff
@@ -69,6 +69,11 @@ protected function build_aspath_regexp($parameter) {
   protected function build_as($parameter) {
     return $this->get_aspath($parameter);
   }
+
```

Comment:
Not required if the IPv6 specific AS path regexp function is removed and its logic included in the existing AS path function.